arch/arm/rules.mk

```diff
@@ -69,6 +69,18 @@ HANDLED_CORE := true
 ENABLE_THUMB := true
 SUBARCH := arm-m
 endif
+ifeq ($(ARM_CPU),cortex-m55)
+GLOBAL_DEFINES += \
+	ARM_CPU_CORTEX_M55=1 \
+	ARM_ISA_ARMv7=1 \
```

Comment:
Will have to probably add proper ARM_ISA_ARMV8 support at some point, but this looks okay, because the compiler has its own intrinsics that CMSIS uses.